apps/src/blockly/utils.js

```diff
@@ -1,5 +1,10 @@
-import {SOUND_PREFIX} from '@cdo/apps/assetManagement/assetPrefix';
 import _ from 'lodash';
+import {SOUND_PREFIX} from '@cdo/apps/assetManagement/assetPrefix';
+
+export function readBooleanAttribute(xmlElement, attribute) {
```

Comment:
Are we always guaranteed that the boolean attribute will be a string?

Reply:
It could also return `null`, but I think that in that case we'd return `false` below. In the case of `usercreated` being missing, that's what we'd want, but it could be worth explaining that case somehow.

Reply:
According to MDN, it should always be a string (possibly empty) or `null`: https://developer.mozilla.org/en-US/docs/Web/API/Element/getAttribute. I can add a comment!